mitxgraders/formulagrader.py

```diff
@@ -410,8 +491,14 @@ def __init__(self, config=None, **kwargs):
         """
         super(FormulaGrader, self).__init__(config, **kwargs)
 
-        # finish validating blacklist/whitelist
+        # finish validating
         validate_blacklist_whitelist_config(self.config['blacklist'], self.config['whitelist'])
+        validate_no_collisions(self.config, keys=['variables', 'numbered_vars', 'user_constants'])
```

Comment:
You need to take `numbered_vars` out of this collision validation, per our discussion.

Reply:
Wait, I thought we wanted to keep that. That ensures
```python
grader0 = FormulaGrader(
    variables=['a'],
    numbered_vars=['a']
)
```
is invalid. That should be invalid, and it's easy to validate. That will give you a non-suppressible error.

In contrast:
```python
grader1 = FormulaGrader(
    variables=['a_{1}'],
    numbered_vars=['a']
)
```
is valid and raises no error (suppressible or otherwise). In an ideal world, grader1 would raise a suppressible error, but we decided that is not worth it.

Reply:
I don't think there's anything wrong with `a` being a variable and a numbered variable, because that allows variables `a` and `a_{n}` in the same problem. It's a bit unusual, but there's no issues internally for us?

Reply:
one final commit?

Reply:
Ugh. I’m not very on top of it today, apparently. 

  I’m away from my computer now, but I will fix it tonight or tomorrow morning.